examples/svm/plot_rbf_parameters.py

```diff
@@ -11,10 +11,13 @@
 'close'. The ``gamma`` parameters can be seen as the inverse of the radius of
 influence of samples selected by the model as support vectors.
 
-The ``C`` parameter trades off misclassification of training examples against
-simplicity of the decision surface. A low ``C`` makes the decision surface
-smooth, while a high ``C`` aims at classifying all training examples correctly
-by giving the model freedom to select more samples as support vectors.
+The ``C`` parameter trades off correct classification of training examples 
+against maximization of the decision function's margin. For larger values of 
+``C``, a smaller margin will be accepted if the decision function is better at 
+classifying all training points correctly. A lower ``C`` will encourage a larger 
+margin, and a more simple decision function, even if that decision function 
```

Comment:
"and a more simple decision function [...]." -> "therefore a simpler decision function, at the cost of training accuracy?

Without the "therefore" it might seem that "simplicity" is a different measure that we're not defining, (and frankly would be tricky to define.) This way we just suggest that the two are related.

The other part of my proposed change is simply reducing verbosity :)